### PR TITLE
buggy snake!

### PR DESCRIPTION
**Pull request by jmou**
This is buggy! There are definitely off-by-ones

### CODE REVIEW

**Comment by jmou** [comment]
@kiddspazz

File: snake.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    RIGHT = (1, 0)
+
+    def __init__(self, width, height):
+        self.width = width
+        self.height = height
+        self.running = True
+        # front of list is head, back of list is tail
+        self.snake = [(self.width // 2, self.height // 2)]
+        self.direction = self.RIGHT
+        self.spawn_apple()
```
Comment (travis-horton) [inline snake.py:71]: does this turn in to a self.apple?

File: snake.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            self.apple = (randint(0, self.width - 1), randint(0, self.height - 1))
+            if self.apple not in self.snake:
+                break
+
+    def quit(self):
+        self.running = False
+
+    def game_over(self):
+        # TODO present game over screen
+        self.running = False
```
Comment (travis-horton) [inline snake.py:85]: maybe add a print "you lost"?

**Comment by jmou** [comment]
@markorodic

**Pull request merged**